### PR TITLE
update download/server/linuxone-signup-thank-you to vbt

### LINKED ISSUE #1641: /download/server/linuxone-signup-thank-you/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update download/server/linuxone-signup-thank-you to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [[page](http://0.0.0.0:8001/download/server/linuxone-signup-thank-you)
* compare to the [design](https://www.dropbox.com/search/work?path=%2F00_web_team%2F_ubuntu.com%2Fnew+projects%2Fvanilla+update%2Fpatterns+inventory&preview=containers-thank-you.png&qsid=21830873026803084826401499240920&query=containers) *based on containers* and the [demo](http://www.ubuntu.com-1641-download-server-linuxone-signup-thank-you.demo.haus/download/server/linuxone-signup-thank-you)

## Issue / Card

Fixes #1641

### CODE REVIEW

**Comment by yaili** [comment]
- [ ] The 3 columns at the bottom of the page are not aligned properly with the heading "More about our enterprise services" above (L screens)
- [ ] The 3 columns at bottom are showing bullet icons in S screens - is it supposed to be a list?

**Comment by pmahnke** [comment]
@yaili fixed... but there is no strip--white in this?

that bottom section was a list... so strange.

**Comment by yaili** [comment]
- [ ] Sorry, can we make the "More about our enterprise services" a p-strip--light?

**Comment by pmahnke** [comment]
@yaili it's light now.

File: static/sass/_v1_pattern_strip.scss [hunk 1]
```diff
@@ -39,6 +43,12 @@
   }
 }
 
+@mixin ubuntu-p-strip-white {
```
Comment (barrymcgee) [inline static/sass/_v1_pattern_strip.scss]: Strips are white by default - why is this needed? toggling it on/off makes no difference to visual of the page?

Comment (pmahnke) [inline static/sass/_v1_pattern_strip.scss]: @barrymcgee I was led to believe that p-strip was light grey?  happy to remove.

Comment (pmahnke) [inline static/sass/_v1_pattern_strip.scss]: @barrymcgee removed
